Template/Tag/VisualWebsiteOptimizerTag.web.js

```diff
@@ -0,0 +1,67 @@
+(function () {
+    var libLoaded = false;
+    var libAvailable = false;
+
+    function loadVwoCode(accountId)
+    {
+        if (libAvailable || typeof window._vwo_code === 'object') {
+            libAvailable = true;
+            libLoaded = true;
+            return;
+        }
+        if (!libLoaded) {
+            
+            libLoaded = true;
+        
+            window._vwo_code = (function() {
+                var account_id = accountId,
+                    settings_tolerance = 2000,
+                    library_tolerance = 2500,
+                    use_existing_jquery = false,
```

Comment:
Where do these Parameters come from and should they maybe be configurable?

Reply:
They are part of the recommended VWO js code. As far as I know, it does not make much sense to change them, but I would keep them this way, so that it is easier to understand what this code does in comparsion to the recommended code.

Reply:
Sounds good to me